runtime/doc/starting.txt

```diff
@@ -529,8 +491,7 @@ X11 GUI support.  See |gui-resources|.
 ==============================================================================
 3. Initialization				*initialization* *startup*
 
-This section is about the non-GUI version of Vim.  See |gui-fork| for
-additional initialization when starting the GUI.
+This section is about the non-GUI version of Vim.
```

Comment:
I think we can take this out.  Even if there are a million neovim UIs we should strive to keep the official UIs consistent.